plugins/BEdita/Core/src/Model/Table/AsyncJobsTable.php

```diff
@@ -268,6 +268,21 @@ protected function findCompleted(Query $query)
     }
 
     /**
+     * Finder for uncompleted async jobs.
+     *
+     * This finder returns a query object that filters asynchronous jobs that haven't been completed yet.
+     *
+     * @param \Cake\ORM\Query $query Query object instance.
+     * @return \Cake\ORM\Query
+     */
+    protected function findUncompleted(Query $query)
```

Comment:
`findPending` is fine :)
Expected (future?) finders could be: `findPending` (or `findActive`), `findInactive`, `findSucceeded`, `findFailed`, `findExpired`